plugins/askrene/test/run-mcf-large.c

```diff
@@ -22,6 +22,10 @@
  * A feasible solution is guaranteed.
  * The last test case has 0 nodes and should be ignored. */
 
+/* check-source does not like me to use scanf, how am I going to read the test
+ * data otherwise? */
+#define myscanf scanf
```

Comment:
LoL.  fscanf :)


But with zlib you'll end up with `gzgets(gzFile file, char *buf, int len);` then use sscanf on the buffer anyway.

Reply:
3f39522075cdb6dc1c4befdd643748b55ba7101f